Cargo.toml

```diff
@@ -15,11 +15,11 @@ readme = "README.md"
 travis-ci = { repository = "TedDriggs/darling" }
 
 [dependencies]
-darling_core = { version = "=0.2.1", path = "core" }
+darling_core = { version = "=0.2.2", path = "core" }
 darling_macro = { version = "=0.2.1", path = "macro" }
```

Comment:
Please bump the version of `darling_macro` and `darling` as well; all 3 crates should have identical version numbers.